conftest.py

```
@@ -17,6 +17,19 @@ class AppLoadError(ImportError):
     pass
 
 
+@pytest.fixture(scope="session", autouse=True)
+def init_test_database():
+    """Initialize test database tables before running tests."""
+    try:
+        from core.db import init_db
+
+        init_db()
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)